### ISSUE TITLE
Is there a way to extract the mask with an API call?

### ISSUE THREAD

**Issue by simohr** [djbielejeski/a-person-mask-generator#22]
Not a bug, rather a question/feature request. Like the title says I have been trying to find a way to get the mask only via API call. Is there a way to do this?

**Comment by monstari** [comment]
Same, I'd like to use it as negative mask.

**Comment by mattc95** [comment]
Yes, but you may need to write an api.py file and put it in the extension folder.
